charts/mobu/values.yaml

```diff
@@ -1,63 +1,66 @@
 # Default values for mobu.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 
-replicaCount: 1
+# -- Autostart specification. Must be a list of mobu flock specifications.
+# Each flock listed will be automatically started when mobu is started.
+autostart: []
 
-image:
-  repository: lsstsqre/mobu
-  tag: 1.1.9
-  pullPolicy: IfNotPresent
+# -- Base URL used to find other services in the environment such as Nublado
+# and TAP
+# @default -- None, must be set
+environmentUrl: ""
 
-mobu_secrets_path: ""
-
-environment_url: ""
+# -- Path to the Vault secret containing the Slack alert hook
+# @default -- None, must be set
+vaultSecretsPath: ""
 
+# -- Override the base name for resources
 nameOverride: ""
+
+# -- Override the full name for resources (includes the release name)
 fullnameOverride: ""
 
-podSecurityContext:
-  runAsNonRoot: true
-  runAsUser: 1000
-  runAsGroup: 1000
+image:
+  # -- mobu image to use
+  repository: lsstsqre/mobu
+
+  # -- Pull policy for the mobu image
+  pullPolicy: IfNotPresent
 
-pull_secret: ""
+  # -- Tag of mobu image to use
+  # @default -- The appVersion of the chart
+  tag: ""
 
-securityContext:
-  allowPrivilegeEscalation: false
-  capabilities:
-    drop:
-      - ALL
-  readOnlyRootFilesystem: true
+# -- Secret names to use for all Docker pulls
+imagePullSecrets: []
```

Comment:
If we are biting the bullet and saying "we will use vault secrets" then let's just create pull-secret by default and stuff it into the default values yaml, is my vote.

Reply:
We aren't doing that anywhere else, so I think that would be a separate change that we should do at the same time across all charts rather than piecemeal. If we do that, we'd have to figure out where to configure the path to the Vault secret for the pull-secret (repeating it in the configuration of every single service isn't attractive).